substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/image/sources/SourceManager.java

```diff
@@ -173,7 +178,7 @@ private static Path computePrototypeName(String fileName, String packageName, So
     }
 
     /**
-     * A whitelist of packages prefixes used to pre-filter JDK runtime class lookups.
+     * An accept list of packages prefixes used to pre-filter JDK11+ runtime class lookups.
      */
     public static final String[] JDK_SRC_PACKAGE_PREFIXES = {
```

Comment:
Stream.concat(Stream.of(JDK8_SRC_PACKAGE_PREFIXES), Stream.of("org.graalvm.compiler")).toArray(String[]::new)